django_migration_linter/sql_analyser.py

```diff
@@ -37,7 +37,8 @@ def has_default(sql, **kwargs):
     {
         "code": "NOT_NULL",
         "fn": lambda sql, **kw: re.search("NOT NULL", sql)
-        and not re.search("CREATE TABLE", sql),
+        and not re.search("CREATE TABLE", sql)
+        and not re.search("(?<=DROP) NOT NULL", sql),
```

Comment:
Could we rather modify the first NOT NULL regex instead? This code wouldn't match this file:

```
ALTER TABLE x ADD COLUMN y NOT NULL;
ALTER TABLE x ALTER COLUMN z DROP NOT NULL;
```

Reply:
Done